evident/tests/support_files/alpha_otu.txt

```diff
@@ -0,0 +1,121 @@
+#SampleID	Observed_OTUs
```

Comment:
after discussing with @serenejiang we thought it will be good to change the sample names to something more real, perhaps qiita-like, something like 10000.000001010(#), where (#) is the current sample name.

Reply:
done